### PR TITLE
Mozilla.Thunderbird  version 102.3.0

### PR DESCRIPTION
**Pull request by TheBossMagnus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82628)

### CODE REVIEW

**Comment by TheBossMagnus** [comment]
no way to bulk update all sha256?

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+- csv
+- eml
+- ldif
+- xpi
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/en-US/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 7F6A5E83254894F680F392545FE79EA8DCFCA7E3BCC075EC0FD876BE6751CDFE
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:23]:
```suggestion
  InstallerSha256: 4BE92CFF5DEA90DABF1F0591DB98A5641A18ED3F4C02AE4B283DECD9C1C58C3B

```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/en-US/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 1B25D36494228B95E86D634CA343BB366B7053A12E697A000D3F4BBD25F1D914
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:29]:
```suggestion
  InstallerSha256: 722E5B8F0F8F71E49CD9A518C9F08A5EC47078C309C289C3C6F2D8D51AAA20DB

```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/en-US/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: DA0B9E634AD67C34F925D307AF648E4CFAAB0AE0ACAB1B6E35A088BD4BB51CE6
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:35]:
```suggestion
  InstallerSha256: DDA1529602232C8EB9478CD8911A3795FEC6F93CB34AAF5F20E0A75E6764F272

```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/en-US/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 96FD9AFE8DCA83A1865497AAB3A1B2F3EF4BE1D347A13BA3B484FA68BE2AE002
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:43]:
```suggestion
  InstallerSha256: C96ACCAD5E011E8F78C58DE102BC9C8D62810700E7F6E6B789BBC2CBE07E84FB

```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: en-GB
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/en-GB/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 550B2BE94981DEDF31B7A112D4595A62BB6BBB523FF8DD2204CAB13787F97EB4
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:51]:
```suggestion
  InstallerSha256: F062041309FE9078C618B7E118AFAA22DCBEC636EA9B2EADF5339B72017582D8

```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: en-GB
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/en-GB/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: A121FBEEB045B868B5BF2B08C75167D4A801762ABEC3565F0FBBB6C87B547928
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:57]:
```suggestion
  InstallerSha256: 7F6A5E83254894F680F392545FE79EA8DCFCA7E3BCC075EC0FD876BE6751CDFE

```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: en-GB
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/en-GB/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: DE7D38CBF628B9658FE036238FCD7AA5698B7B110FF1ADD370126ABDF0352387
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:63]:
```suggestion
  InstallerSha256: E20AB267EAC8DE354ADEC7C0169497B72815D3C8D72FF78F319B5C936E71F7CF

```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: en-GB
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/en-GB/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: F0721B0A942D193ADE831B40490F7B41ED6178DE4DB87AA08A54E44003583083
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:72]:
```suggestion
  InstallerSha256: 1DBBCEDB47E52F1D1453D8D87881B61DE2EDAD1E6581000D650AD5539BD84AA1
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/de/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: C72E8C888671794205C1296437F7E6A98FAAD69F5D194F43882B86C803851C9D
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:80]:
```suggestion
  InstallerSha256: 52B8E4555C18682CFAEC377E5D7134893FA9CF026EB96965E66030C5CD57CEB7
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: de-DE
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/de/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: FD2287F4F3C0FC365C8D0F3A5D5C9E75DC39B2ECF9C209698405D7A3381CCB0B
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:86]:
```suggestion
  InstallerSha256: 65900BFB64AF2565DFA2AFDF92EABEFAC87E21A6FB64326824A73BF78EF58EC0
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/de/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: C5EF18294E4503D378B3E38030F5DFB830178DDD77120BAD75A6B0BE2B6DEC40
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:92]:
```suggestion
  InstallerSha256: 296241DAAE675603FEDDCEE2AD0EF0F2A463A8DB5FED400CBD1B7CBF61D227AD
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 12]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: de-DE
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/de/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 722218D8D6798DD09097DDF234DA5E7B1DC5D596ED87736A00C377549DC66D57
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:100]:
```suggestion
  InstallerSha256: 406B990E550CAC899F4229AD745669E977CFCB4CF8F7A114A69C5CCD0AF1C540
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 13]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: zh-CN
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/zh-CN/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: CBB8A2F84DC3BC443B120C87D10A7CB98B4C988485554E419818321EE22AF1B8
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:108]:
```suggestion
  InstallerSha256: 10D4862A9783A1482CA8B530499A4DEEC9A52D17441E3836990347DBFD375C59
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 14]
```diff
@@ -0,0 +1,245 @@
...
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/zh-CN/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 47D54FCE679DCF5A169074D02376FBB3B92D8509D171A4E977DCB4454923603B
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: zh-CN
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/zh-CN/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: ED1F55B3A4CB989C6E382784D1252BB6AD89AF149369D16807CA23470AD8B446
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:120]:
```suggestion
  InstallerSha256: 3508480DC26D28BE0972C0E9E3A4D338B2F338B91F98C6CEBC1A98374DF58966
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 15]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/zh-CN/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 7B5E833DB32DBE129DE7C95BA0422DF6F5A13726054E80805AD7F8110EFA6AB7
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:128]:
```suggestion
  InstallerSha256: 17F0CFB1DB308647D27CB824C327B87D02211341875DA0ECF9F405476342FA4A
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 16]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: zh-TW
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/zh-TW/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 03D653962E31B5E68EFBF4D008765E3EAC782FB8503F65C35ED6FB9F939E2711
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:136]:
```suggestion
  InstallerSha256: 4BACE2CE3754EB323DE9240248087BF9CFD85104C0B394F0105169C9E637497B
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 17]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: zh-TW
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/zh-TW/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 63E60C24E2DE3F043A30CFE002A9663DA0CF95CB2F78C90EEE83403E432D1918
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:142]:
```suggestion
  InstallerSha256: 796098D7EAB261325DD534ECBDE9FAFC8D2E060B5DFA014ABC7BDA202021CEC9
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 18]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: zh-TW
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/zh-TW/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 043CA470DC5EC2A3985CBB78DC796131F2BD95068799CFF5FB6F8FFAFC4CEAA5
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:148]:
```suggestion
  InstallerSha256: B4DAFD535FA4403BE4F937222BA44D4314F5DF734360E580D2EBF0D5A3C23635
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 19]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: zh-TW
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/zh-TW/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 1663C01D5A4F884DC6AB8E7D10F962DBF5D38B05AB95E8B0F5E26B6A1FF73359
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:156]:
```suggestion
  InstallerSha256: 0BB5D4FB4F5E105995C03F52EDB91CD43D2D694D7990E12A7CFAC65A224C23A2
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 20]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: es-ES
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/es-ES/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: C468880A8EA958AC95845468494332615827E96BE193324011063B7FFB656713
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:164]:
```suggestion
  InstallerSha256: 8262B3D703B0F2CF9F1CAC1949406438AF5C929CD673B105752DC77E4080FBD4
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 21]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: es-ES
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/es-ES/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 766F84EF050FCBD0BADAA137669510F7483C4FFD7D6801AB65A95D51989B264F
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:170]:
```suggestion
  InstallerSha256: 73F8C8D322F4B953DC2B1FD94200E3726353B40315011FD5133291CB2F15D3BE
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 22]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: es-ES
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/es-ES/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: CA68F05D88179520AD23FF6353B0D3AFBFA2CA6FE3532D75251162759A50D4EC
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:176]:
```suggestion
  InstallerSha256: 80B5D9E747EC615E60A05A178AA36B74B3AFB6B563E921B43A40E670A006147D
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 23]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: es-ES
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/es-ES/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 47A278B26C333051ED636B9253DE2489376BDFE16B96DAEC5CA4F2BDACC82804
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:184]:
```suggestion
  InstallerSha256: 0C7F7F2713B7BAD5E521132C19C022087BEE94A8A335A18D9F91B616BCC48DE9
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 24]
```diff
@@ -0,0 +1,245 @@
...
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/ja/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: C686381B1ABBC0B11D0E95B9A191720DADB11ABF6FD7918AB81E747E3D1C8452
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: ja-JP
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/ja/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 956F68055498B975513D38AD005379D679D388DC95761FF1D2330AD3D8C668AA
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:198]:
```suggestion
  InstallerSha256: F3914F93A1FFA2041F4588BD960414A36338676C911EE1BEB57ACBEF1D396852
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 25]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: ja-JP
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/ja/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 3C369AD21EB90C309F9CEB0A9AC7FE539818E2FA63A3757229789C458AD5C47A
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:204]:
```suggestion
  InstallerSha256: 40FF9CDB75815D278315DA26DAB8678F6F2B1C12E3CE7F600B875AE18C33FF9A
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 26]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: ja-JP
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/ja/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: D305F2F2EA9904EB19C9696E09BD6451F6E21A4A1F5F76CFA008789428EFC216
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:212]:
```suggestion
  InstallerSha256: 9830F953548BF2A924C87879667B6014E6E147E4A55ED17CFF63EE894818CDD4
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 27]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: fr-FR
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/fr/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 00DD26354B0A865578D70B18ED1BD842C6836F957B47377C181B686724131679
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:220]:
```suggestion
  InstallerSha256: 2790CEBB120F0BE89AF59FC704770A59054385487BD40B5B6507BF624EA56CE6
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 28]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: fr-FR
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/fr/Thunderbird%20Setup%20102.3.1.msi
+  InstallerSha256: 0AB0BE075C0EED3CC9852B2529CC27C2F47B1E1AB2B3A969BC1A7BDB2C917C1F
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:226]:
```suggestion
  InstallerSha256: B3FF2B0AE0E9BBB2715A6E401F9A872BA5D8E212BAB0FC455F5DFE9B225A5975
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 29]
```diff
@@ -0,0 +1,245 @@
...
+  ProductCode: '{18D4D2F4-B394-48A2-8886-9C68180CEAD5}'
+- InstallerLocale: fr-FR
+  Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win64/fr/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 6F560EC34D202E5853A1241B76935677AB6EACA6AFCAE058AF908B3BCAE6FA24
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:232]:
```suggestion
  InstallerSha256: C303F930397ACA8EC2551F40F6A468F6E00C4FDE0465E8C511D37CE041C1D29D
```

File: manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml [hunk 30]
```diff
@@ -0,0 +1,245 @@
...
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: fr-FR
+  Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/102.3.1/win32/fr/Thunderbird%20Setup%20102.3.1.exe
+  InstallerSha256: 8F7A1AE57F914CA2A4136AD1A5062580D50A18EF1DC038D20CE0B996E6D8D5DD
```
Comment (stephengillie) [inline manifests/m/Mozilla/Thunderbird/102.3.1/Mozilla.Thunderbird.installer.yaml:240]:
```suggestion
  InstallerSha256: E6627326D797913338494376E6D848C153527DE11C2CDC266868C4CD8ECE1F0D
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Please verify some or all of these updated hashes.

**Pull request closed without merging**